Vue Component.sublime-syntax.yaml-macros

```diff
@@ -101,11 +115,18 @@ contexts: !merge
         - tag-generic-attribute-meta
         - tag-generic-attribute-value
 
-  script-common: !prepend
+  script-common:
+    - meta_prepend: true
     - include: script-lang-attribute
 
+  script-javascript-content:
+    - match: (?=\S)
+      embed: scope:source.jsx
```

Comment:
Only having the `source.jsx` scope here also scares me due to being likely a breaking change.

Many packages might rely on seeing `source.js` here and triggering specific functionality based on that.

My custom keybindings for example rely on having `source.js` here as.

Reply:
There is [a section on JSX on the Vue 2.0 docs](https://vuejs.org/v2/guide/render-function.html#JSX) where it is mentioned as being supported through a plugin, however I have never seen JSX being used in a Vue single-file component (which albeit isn't that much, but it tells something). Since this wasn't supported before and is a breaking change, I suggest not to use JSX here unless someone requests it.

Reply:
I'd also weigh in that we should not break any compatibility and set the context as `source.js` here.

I would like to maintain some level of parity with Vetur (the VS Code syntax file) and that appears to use `source.js` as well (however JSX syntax seems to work out of the box with that scope).

Reply:
I've gone back to JS.